src/main/java/nl/hsac/fitnesse/util/iban/BGIbanGenerator.java

```diff
@@ -0,0 +1,50 @@
+package nl.hsac.fitnesse.util.iban;
+
+/**
+ * Created by Koen on 20-4-2017.
+ */
+public class BGIbanGenerator extends IbanGenerator {
```

Comment:
Does this generator actually work? Should there be a test for it, or should it be removed?